k8s/base/deployment.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: zitadel
+  labels:
+    app: zitadel
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: zitadel
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: zitadel
+        version: v1
+    spec:
+      containers:
+      - name: zitadel
+        image: docker.pkg.github.com/caos/zitadel/zitadel:latest
+        imagePullPolicy: Always
+        args: ["-login=false", "-console=false"]
```

Comment:
we should no supply this as standard args

Reply:
master does currently not proceed without these flags. once master is solid we will remove it anyway.

Reply:
in which PR will this happen @livio-a @fgerschwiler? because we should reference it for further tracking